docs/dev/logger.md

```diff
@@ -0,0 +1,52 @@
+### Logging
+
+Logging utilizes the [`github.com/cihub/seelog`](https://github.com/cihub/seelog) package as its underlying framework. 
+It is accessible via `pkg/util/log` and the `comp/core/log` component wrapper.
```

Comment:
```suggestion
You can access logging through `pkg/util/log` and the `comp/core/log` component wrapper.
```

Reply:
`wrapper` -> `wrappers` from the suggestion